common/app/model/content.scala

```diff
@@ -376,7 +377,8 @@ object Article {
       iosType = Some("Article"),
       javascriptConfigOverrides = javascriptConfig,
       opengraphPropertiesOverrides = opengraphProperties,
-      twitterPropertiesOverrides = twitterProperties
+      twitterPropertiesOverrides = twitterProperties,
+      isMinute = content.tags.isUSMinuteSeries
```

Comment:
A bit inconsistent here, is it `USMinute` or is it `Minute`? What will it be in the future?